### PR TITLE
Update User Guide with GFMD

### PR DESCRIPTION
**Pull request by ivanleekk**
Update User Guide to use GFMD

Add mockup images from Figma to the repo

### CODE REVIEW

File: docs/images/Add.png
Comment (jingting1412) [inline docs/images/Add.png:1]: Maybe add additional line: “Data saved to file” below the current message according to the UG

File: docs/images/Delete.png
Comment (jingting1412) [inline docs/images/Delete.png:1]: Similar to the add, maybe can add “Data saved to file” or like “Data updated” below the current message similar to what is stated in the UG

File: docs/images/Edit.png
Comment (jingting1412) [inline docs/images/Edit.png:1]: Similar to previous comments

File: docs/UserGuide.md
Comment (jingting1412) [inline docs/UserGuide.md:1]: The find and save part looks good to me. Good job!

**Review by craigtonlian — CHANGES_REQUESTED** [review]
Consider updating your branch name from `Update_User_Guide` to `update-user-guide` according to the convention guide in https://se-education.org/guides/conventions/git.html :blush:

**Pull request closed without merging**